master/buildbot/klog.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+import json
+
+
+def __get_json(failure_):
+    failure_dict = {
+        'type': str(type(failure_.type())),
+        'value': str(failure_.value),
+        'msg': str(failure_),
+        'datetime': str(datetime.datetime.now()),
```

Comment:
I looked at the current code and we are dumping json logs in three places. All of them have a `name` and `description` to make it easier to identify them in the logs.

Could you add a `name = 'exception'`, `description = 'Unhandled exception'` to this dict?

In the future it would be nice to make a common function for all our json log outputs (something that you would pass the dict + name + description), but we don't need to do that now.